utilities/tools/AzureDevOps/Register-AzureDevOpsPipeline.ps1

```diff
@@ -126,9 +113,6 @@ function Register-AzureDevOpsPipeline {
         [ValidateSet('gitHub', 'tfsgit')]
         [string] $SourceRepositoryType = 'gitHub',
 
-        [Parameter(Mandatory = $false)]
```

Comment:
Does the corresponding Wiki page in `docs\wiki\Interoperability - Register Azure DevOps Pipelines.md` need to be updated as well?

Reply:
Surprisingly, no. The PAT was never used